prow/github/types.go

```diff
@@ -1453,3 +1453,50 @@ type DirectoryContent struct {
 	Name string `json:"name"`
 	Path string `json:"path"`
 }
+
+// WorkflowRunEvent holds information about an `workflow_run` GitHub webhook event.
+// see // https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#workflow_run
+type WorkflowRunEvent struct {
+	Action       string       `json:"action"`
+	WorkflowRun  WorkflowRun  `json:"workflow_run"`
+	Workflow     Workflow     `json:"workflow"`
+	Repo         *Repo        `json:"repository"`
```

Comment:
Why name this differently than in the json?

Reply:
We do the same in the rest of the event structs. I just kept the same approach.